src/activateDaffodilDebug.ts

```diff
@@ -51,10 +66,9 @@ export function activateDaffodilDebug(context: vscode.ExtensionContext, factory?
 		})
 	);
 
-	context.subscriptions.push(vscode.commands.registerCommand('extension.dfdl-debug.getProgramName', config => {
+	context.subscriptions.push(vscode.commands.registerCommand('extension.dfdl-debug.getDataName', config => {
 		return vscode.window.showInputBox({
```

Comment:
Is there any way to add a file picker to this?  Typing full paths could be tedious if files are not in the workspace, which most likely they won't be.